lib/login.js

```diff
@@ -43,23 +43,23 @@ var Login = module.exports = {
             }
           })
       })
-      .spread(function (res, body) {
+      .spread((res, body) => {
         if (res.statusCode !== 201) {
           throw new Error('we could not generate a token')
         }
         if (!body.token) {
           throw new Error('we did not get a token back. :(')
         }
-        return Promise.fromCallback(function (callback) {
-          args._user.githubLogin(body.token, callback)
+        return Promise.fromCallback((callback) => {
+          Login.user.githubLogin(body.token, callback)
         })
       })
-      .then(function () {
+      .then(() => {
         Login._output('Authenticated!')
       })
-  }),
+  }
 
-  _makeRequest: Promise.method(function (creds, token) {
+  static _makeRequest (creds, token) {
     var githubURL = process.env.RUNNABLE_GITHUB_URL || 'https://api.github.com'
     var parsedURL = url.parse(githubURL)
```

Comment:
`const` since the returned object is being mutated but no assignment is being made, or `let` because a mutation is taking place. Which do we want to do here?